extension/content_script.js

```diff
@@ -274,20 +280,17 @@ Highlight portion pointed to by tracker, and unhighlight previous sentence (if n
 */
 function highlight(sentenceId) {
 	let sentenceStyle = trackerStyle.getSentenceStyle();
+	let keywordStyle = trackerStyle.getKeywordStyle(); 
+
 	// Unhighlight if previous highlight already exists.
 	if (highlightedSentenceId !== null) {
 		doc.getSentenceEls(highlightedSentenceId).removeClass(sentenceStyle);
+		doc.getSentenceKeywordsEls(highlightedSentenceId).removeClass(keywordStyle)
 	}
-
 	// Highlight the sentence.
 	doc.getSentenceEls(sentenceId).addClass(sentenceStyle);
-
-	// Highlight they keywords.
-	let sentencePtr = doc.getSentence(sentenceId);
-	let container = doc.getContainer(sentencePtr.containerId);
-	let start = sentencePtr.start;
-	let end = sentencePtr.end;
-	highlightKeyWords(container, start, end);
+	// Highlight the keywords. 
+	doc.getSentenceKeywordsEls(sentenceId).addClass(keywordStyle)
```

Comment:
aaaah looks so nice now :)

-- and semicolon

Reply:
Hahaha